web/package.json

```diff
@@ -71,8 +71,8 @@
     "cypress": "start-server-and-test start:cypress 3030 cy:open",
     "cypress:ci": "start-server-and-test start:cypress 3030 cy:run",
     "eject": "react-scripts eject",
-    "lint": "eslint './src/**/*.{ts,tsx}'",
-    "lint:fix": "eslint --fix './src/**/*.{ts,tsx}'",
+    "lint": "eslint",
```

Comment:
I'm assuming this change widens the file types that are linted? So typescript, tsx, javascript, etc?

Reply:
whoops. i made comments on last pr and didn't move them. one sec.

Reply:
moved all to eslint.config to keep it in one place